site/content/docs/4.3/components/navs.md

```diff
@@ -245,6 +247,8 @@ If you're using navs to provide a navigation bar, be sure to add a `role="naviga
 
 Note that navigation bars, even if visually styled as tabs with the `.nav-tabs` class, should **not** be given `role="tablist"`, `role="tab"` or `role="tabpanel"` attributes. These are only appropriate for dynamic tabbed interfaces, as described in the [<abbr title="Web Accessibility Initiative">WAI</abbr> <abbr title="Accessible Rich Internet Applications">ARIA</abbr> Authoring Practices](https://www.w3.org/TR/wai-aria-practices/#tabpanel). See [JavaScript behavior](#javascript-behavior) for dynamic tabbed interfaces in this section for an example.
 
+On the other hand, the `aria-current` attribute is not necessary on dynamic tabbed interfaces since our JavaScript handles the selected state by adding `aria-selected="true"` on the active tab.
```

Comment:
"On the other hand, " feels a bit out of place here. What's the one hand? i.e. this would make sense if in the previous sentence you'd just said "you must use `aria-current`. 

Anyway, I think it would make most sense to just start the sentence with "The `aria-current` attribute is not necessary...". could even add it to the end of the previous paragraph, rather than starting a new paragraph.